test/test_arhp_wrong_db.rb

```diff
@@ -9,6 +9,10 @@ def setup
     ActiveRecordHostPool::PoolProxy.class_variable_set(:@@_connection_pools, {})
```

Comment:
Sometimes it’s in the setup, sometimes it’s in the teardown. Could be neat if it was one or the other, but not a blocker for this PR.

Reply:
I'm not sure why it was added as a setup in `test_wrong_db` tbh